jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -760,7 +780,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
                                                        NULL,
                                                        NULL,
                                                        was_enumerable,
-                                                       was_configurable);
+                                                       is_current_configurable);
```

Comment:
As I tracked it down: by the time we reach this statement, we know: 
- `is_property_desc_generic_descriptor == false`
- `is_property_desc_data_descriptor != is_current_data_descriptor`
- `is_current_data_descriptor == true`

The last two implies that `is_property_desc_data_descriptor == false`. From this and the first item of the above list we can also infer that `is_property_desc_accessor_descriptor == true`. And actually _that's_ why we are calling `ecma_create_named_accessor_property` here. That's why I think that `if (is_property_desc_accessor_descriptor)` would be the right predicate. IMHO.